rb/spec/tests.bzl

```diff
@@ -0,0 +1,85 @@
+load("@rules_ruby//ruby:defs.bzl", "rb_test")
+load("//common:browsers.bzl", "chrome_data")
+
+ENV = select({
+    "//rb/spec/integration:chrome": {
+        "WD_REMOTE_BROWSER": "chrome",
+        "WD_SPEC_DRIVER": "chrome",
+    },
+    "//rb/spec/integration:edge": {
+        "WD_REMOTE_BROWSER": "edge",
+        "WD_SPEC_DRIVER": "edge",
+    },
+    "//rb/spec/integration:firefox": {
+        "WD_REMOTE_BROWSER": "firefox",
+        "WD_SPEC_DRIVER": "firefox",
+    },
+    "//rb/spec/integration:ie": {
+        "WD_REMOTE_BROWSER": "ie",
+        "WD_SPEC_DRIVER": "ie",
+    },
+    "//rb/spec/integration:safari": {
+        "WD_REMOTE_BROWSER": "safari",
+        "WD_SPEC_DRIVER": "safari",
+    },
+    "//rb/spec/integration:safari-preview": {
+        "WD_REMOTE_BROWSER": "safari-preview",
+        "WD_SPEC_DRIVER": "safari-preview",
+    },
+    "//conditions:default": {},
+}) | select({
```

Comment:
`|`? Interesting. I normally use `+`. Had no idea this would work

Reply:
`+` only works for lists, while `|` works for dicts
https://github.com/bazelbuild/starlark/issues/14